src/core/lv_obj_tree.c

```diff
@@ -623,7 +623,7 @@ static lv_obj_tree_walk_res_t dump_tree_core(lv_obj_t * obj, int32_t depth)
         for(uint32_t i = 0; i < obj->spec_attr->child_cnt; i++) {
             res = dump_tree_core(lv_obj_get_child(obj, i), depth + 1);
             if(res == LV_OBJ_TREE_WALK_END)
```

Comment:
Let's remove `res` and make `dump_tree_core` returns `void`.

Reply:
okay, please review second patch.